weblate/billing/models.py

```diff
@@ -203,10 +205,15 @@ class Meta:
     def __str__(self) -> str:
         projects = self.projects_display
         owners = self.owners.order()
+        owners_visible_names_tuples = list_to_tuples(
+            [x.get_visible_name() for x in owners]
+        )
         if projects:
             base = projects
         elif owners:
-            base = ", ".join(x.get_visible_name() for x in owners)
+            base = format_html_join_comma(
+                "{}", (x for x in owners_visible_names_tuples)
+            )
```

Comment:
The additional comprehension here is not needed at all:

```suggestion
            base = format_html_join_comma(
                "{}", owners_visible_names_tuples
            )
```


But keeping it fully inline might be even easier to understand:

```suggestion
            base = format_html_join_comma(
                "{}", list_to_tuples(x.get_visible_name() for x in owners)
            )
```